src/common/com/intellij/plugins/haxe/ide/module/HaxeModuleBuilder.java

```diff
@@ -51,6 +74,75 @@ public void moduleCreated(@NotNull Module module) {
     final CompilerModuleExtension model = (CompilerModuleExtension)CompilerModuleExtension.getInstance(module).getModifiableModel(true);
     model.setCompilerOutputPath(model.getCompilerOutputUrl());
     model.inheritCompilerOutputPath(false);
+
+    final Project project = module.getProject();
+    List<Pair<String, String>> sourcePaths = getSourcePaths();
+    String srcPath = null;
+
+    if ((sourcePaths.size() > 0)) {
+      srcPath = sourcePaths.get(0).getFirst();
```

Comment:
I think it is more friendly, because you would have to create Main class/method/file anyway (at least I do that, especially when I am developing this=) there are should be a way to create unit tests for it )
If a warning then should be QuickFix